feature-utils/poly-import/src/importer.js

```diff
@@ -2,6 +2,14 @@ import { Status, statusTypes } from "../utils/status";
 import { Telemetry } from "../utils/performance-telemetry";
 import { ZipFile } from "./storage";
 
+export class Importer {
```

Comment:
Is this being used anywere, or supposed to be?

Reply:
Yep, it's going to be an abstract base class eventually.